### PR TITLE
Here is my solution

### PR DESCRIPTION
**Pull request by elliehendersonn**
I think this all works.

### CODE REVIEW

File: bobs_change.py [hunk 1]
```diff
@@ -33,5 +33,20 @@ def run_tests():
 
 # Edit this code:
 def bobs_change(money):
+   
+    if money < 50:
+        print ("Sorry Bob, you don't have enough")
+
+
+    elif money == 50:
```
Comment (andrewcharlton) [inline bobs_change.py]: Do you need these two lines?

**Comment by andrewcharlton** [comment]
Looks good so far but you still need to fix six_or_bust

**Comment by elliehendersonn** [comment]
Sorry sir for not doing six or bust, could you explain it to me tomorrow if
that's alright?

**Comment by andrewcharlton** [comment]
Have a quick look at the problem file - it should be an easy one!

**Pull request closed without merging**